plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibrariesInPluginXmlTest.java

```diff
@@ -52,16 +46,14 @@
   private static final String JSP_API_LIBRARY_ID = "jsp-api";
 
   @Test
-  public void testThereAreExactlyFiveLibraries() {
-    IConfigurationElement[] configurationElements = RegistryFactory.getRegistry()
-        .getConfigurationElementsFor("com.google.cloud.tools.eclipse.appengine", "libraries",
-            "com.google.cloud.tools.eclipse.appengine.libraries.defaultapis");
-    assertThat(configurationElements.length, is(5));
+  public void testLibrarySize() {
+    assertThat(AppEngineLibraries.getLibraries("appengine").size(), is(3));
```

Comment:
Defining the group names as constants in AppEngineLibraries could be useful reduce possible typos.